dockers/docker-fpm-frr/base_image_files/TSB

```diff
@@ -8,4 +16,10 @@ then
 fi
 
 /usr/bin/TS TSB
-echo "Please execute 'config save' to preserve System mode in Normal state after reboot or config reload"
+if [[ "$(sonic-cfggen -d -v DEVICE_METADATA.localhost.type)" == *"SpineRouter"* ]] ; then
+  if [[ "$1" != "chassis" ]] ; then
```

Comment:
a little bit confused here, what's the usage of parameter "chassis" for this script?

Reply:
'chassis' parameter is set when command is triggered from supervisor. 

It is mainly to identify if the command was triggered from supervisor or directly on LC, in order to display the right message to save config. If cmd is triggered from supervisor, the message suggests to use 'rexec all -c config save' (to save config on all LCs), whereas if it is triggered from LC, message suggests to do 'config save' directly on that LC